XenAdmin/Dialogs/AttachUsbDialog.cs

```diff
@@ -75,6 +96,31 @@ private void BuildList()
             }
         }
 
+        private void treeUsbList_SelectedIndexChanged(object sender, EventArgs e)
+        {
+            UsbItem item = treeUsbList.SelectedItem as UsbItem;

```

Comment:
I think this whole method can be implemented in one line as:

```
buttonAttach.Enabled = treeUsbList.SelectedItem is UsbItem;
```

Reply:
Accepted.